tests/test_cacheutils.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
-
+import abc
 import string
+from abc import abstractmethod, ABCMeta
```

Comment:
You only use `abc.ABCMeta`, so importing `ABCMeta` here is unnecessary.  Alternatively, use unqualified `ABCMeta` and remove the unnecessary `import abc`.

```suggestion
from abc import abstractmethod
```

Reply:
Good catch! Removed the unnecessary import, thank you. =)